Controller/ElasticsearchController.php

```diff
@@ -863,11 +820,10 @@ public function searchAction(Request $request)
                 $body['aggs']['agg_' . $id] = $option->getConfigDecoded();
             }
 
-
-            $params['body'] = $body;
+            $searchRequest->setBody($body);
```

Comment:
Here you are setting the body, but this is "too late" for the other call! and we are in another function :p